### PR TITLE
Merge CSS conflict stuff

### PR DESCRIPTION
**Pull request by Sbijok**
Our CSS should now be up-to-date...

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,8 +5,13 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <meta http-equiv="X-UA-Compatible" content="ie=edge">
         <!-- Bootstrap CSS -->
+<<<<<<< HEAD
```
Comment (jerrylammel) [inline index.html:8]: hi Sam, please resolve this conflicts manually

File: index.html [hunk 2]
```diff
@@ -5,8 +5,13 @@
...
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous">
+        <link href="style.css" rel="stylesheet">
+=======
         <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous"> 
         <link href="style.css" type="text/css" rel="stylesheet">
+>>>>>>> main
```
Comment (jerrylammel) [inline index.html:14]: please clean up this

File: style.css [hunk 3]
```diff
@@ -11,6 +11,23 @@ html {
     font-family: Helvetica, sans-serif;   
     text-align: center; 
 }
+.banner-container h2 {
+    font-size: xx-large;
+    text-align: center;
+    background-color: #5471ab;
```
Comment (jerrylammel) [inline style.css:17]: please use css variables defined for backgroud color instead

File: style.css [hunk 4]
```diff
@@ -11,6 +11,23 @@ html {
...
+
+.banner-container img {
+    width: 100%;
+    padding: 0%;
+    margin: 0;
+    border-width: 1px;
+    max-width: 100%;
+    background-color: #5471ab;
+    height: auto;
+    background-image: url("images/image_banner.png");
```
Comment (jerrylammel) [inline style.css:29]: please move images into folder resources/images/

**Review by jerrylammel — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Sbijok) [reply index.html:8]: Ok this is done as per earlier :)

File: index.html (hunk 2 above)
Comment (Sbijok) [reply index.html:14]: Done :)

File: style.css (hunk 4 above)
Comment (Sbijok) [reply style.css:29]: I can't do this until I have the version with the folder in it

Comment (jerrylammel) [reply style.css:29]: well, you can create the folder, sorry for misleading

**Comment by jerrylammel** [comment]
We can improve this in the next session

**Pull request merged**